charts/camunda-platform-alpha/values.yaml

```diff
@@ -529,7 +529,7 @@ console:
     automountServiceAccountToken: false
 
   ingress:
-    ## @param console.ingress.enabled if true, an ingress resource is deployed with the Console deployment. Only useful if an ingress controller is available, like nginx.
+    ## @param console.ingress.enabled if true, an ingress resource is deployed with the Console deployment. Only useful if an ingress controller is available, like nginx. Warning, separated ingress is deprecated and will be removed in the next release, please use global.ingress instead.
```

Comment:
Warning: Separated ingress is deprecated and will be removed in the next release. Please, use Combined ingress by setting global.ingress: true. For more information, see https://docs.camunda.io/docs/self-managed/setup/guides/ingress-setup/